test_util/helpers.py

```diff
@@ -23,81 +24,139 @@ def path_join(p1, p2):
 
 class DcosUser:
     """A lightweight user representation."""
-    def __init__(self, auth_json):
+    def __init__(self, auth_json: dict):
```

Comment:
@jgehrcke I agree with you, however, I notice in a lot of our code, people tend towards labeling dicts that are derived from or intended to be JSON with _json...